src/open_inwoner/templates/admin/catalogusconfig_change_list.html

```diff
@@ -0,0 +1,11 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_urls %}
+
+{% block object-tools-items %}
+{{ block.super }}
+<li>
+    <a href="{% url 'admin:upload_zgw_import_file' %}" class="addlink">
+        {% trans "Create from json export" %}
```

Comment:
```suggestion
        {% trans "Import from file" %}
```
"json export" is too specific IMHO.